model-optimizer/mo/front/common/partial_infer/utils.py

```diff
@@ -126,8 +269,20 @@ def get_shape_from_slice(input_shape: np.ndarray, slices: List) -> np.ndarray:
                 output_shape.append(input_shape[in_idx])
                 in_idx += 1
         else:
-            raise Exception('Element type of a slice List is unacceptable. '
-                            'Allowed types are: Ellipsis, slice, int, and None. Instead got: '. format(type(s)))
+            raise Exception('Element type of a slice list is unacceptable: "{}"'.format(type(s)))
```

Comment:
Because the list of types increased and listing all of them will be too long, so I have decided to list all of them. Also it is a bit useless to list all of them because it does not make sense for the user.